src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1

```diff
@@ -74,7 +74,36 @@ CmdletsToExport = '*'
 VariablesToExport = '*'  
   
 # Aliases to export from this module  
-AliasesToExport = @()
+AliasesToExport = @(
```

Comment:
When i didnt include these, aliases were not working, after this it started working.